src/utils/Processing.ts

```diff
@@ -240,6 +241,29 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+export function checkNullValuesOnArray(resource: any, parentName = ''): void {
+  const resourceName = parentName ? parentName : resource.id ?? resource.name;
+  for (const property in resource) {
+    // We'll want to exclude properties beginning with an "_" as those are arrays containing extensions
+    // corresponding to a primitive array, per the FHIR spec
+    if (property.startsWith('_')) continue;
+    if (_.isPlainObject(resource[property]))
+      // If we encounter an object property, we'll want to check its properties as well
+      checkNullValuesOnArray(resource[property], resourceName);
+    if (Array.isArray(resource[property])) {
+      const nullIndexes: number[] = [];
+      resource[property].forEach((element: any, index: number) => {
+        if (element === null) nullIndexes.push(index);
+      });
+      if (nullIndexes.length > 0) {
+        logger.warn(
+          `The array '${property}' in ${resourceName} is missing values at the following indices: ${nullIndexes}`
```

Comment:
If the problem was in a deeply nested object, then `${property}` is only going to be the property name within the deeply nested object -- but I think it needs to be the path, right?  E.g.,
```json
{
  "resourceType": "Patient",
  "name": [ { "given": [ "Bob", null, "Robert" ] } ]
}
```
Would say:
> The array 'given' in Patient is missing values at the following indices: 1

But it probably should say:
> The array 'name[0].given' in Patient is missing values at the following indices: 1